app/src/main/java/edu/uc/zhukv/droneradarmap/MapsActivity.kt

```diff
@@ -54,9 +71,11 @@ class MapsActivity : AppCompatActivity(), OnMapReadyCallback {
                 setUpMap()
             }
         }
-        val mapFragment = supportFragmentManager
-                .findFragmentById(R.id.map) as SupportMapFragment
-        mapFragment.getMapAsync(this)
+        //Changed to .also for less redundancy see: https://kotlinlang.org/api/latest/jvm/stdlib/kotlin/also.html
```

Comment:
To prevent redundancy and more clear code I replaced the self-reference with "it" and included a .also function to allow the "it" use.

Reply:
For a single line, .also is a bit of overkill, unless it is being used to avoid  a NullPointerException (which is not the case here).

Also, curlies are not tabbed properly.